src/EPR.PRN.Backend.Data/Repositories/ObligationCalculationRepository.cs

```diff
@@ -2,26 +2,18 @@
 using EPR.PRN.Backend.Data.Interfaces;
 using Microsoft.EntityFrameworkCore;
 
-namespace EPR.PRN.Backend.Data.Repositories
+namespace EPR.PRN.Backend.Data.Repositories;
+
+public class ObligationCalculationRepository(EprContext context) : IObligationCalculationRepository
```

Comment:
If we are going to use a primary constructor here then we need to keep it consistent across the solution

Reply:
Yes, that's suggestion from Visual Studio / GitHub. Hence, changed